modules/cli/src/main/scala/scala/cli/commands/ScalacLikeCommand.scala

```diff
@@ -0,0 +1,52 @@
+package scala.cli.commands
+import scala.build.compiler.SimpleScalaCompiler
+import scala.build.options.{BuildOptions, Scope}
+import scala.cli.commands.util.SharedOptionsUtil._
+
+/** Trait to mixin for commands which are intended to mimic `scalac` behaviour, allowing for
+  * printing scalac help & other options, even without passing input sources. Meant as an extension
+  * of [[scala.cli.commands.ScalaCommand]]
+  *
+  * @tparam T
+  *   command options type
+  */
+trait ScalacLikeCommand[T] { self: ScalaCommand[T] =>
```

Comment:
Instead of creating this trait, couldn't `maybePrintSimpleScalacOutput` be added directly to `ScalaCommand`, and have a signature like
```scala
def maybePrintSimpleScalacOutput(buildOptions: BuildOptions): Unit
```